http/CHANGELOG.md

```diff
@@ -2,6 +2,151 @@
++twilight_http::routing::Route::DeleteMessageSpecificReaction
+
+-twilight_http::request::channel::invite::CreateInvite::target_user
++twilight_http::request::channel::invite::CreateInvite::target_user_id
+
+-twilight_http::request::channel::invite::CreateInvite::target_user_type
++twilight_http::request::channel::invite::CreateInvite::target_type
+```
+
+`UpdateStageInstance` requests now look like this:
+
+```diff
+-client.update_stage_instance(channel_id, topic)?.await?;
++client.update_stage_instance(channel_id)
++    .topic(topic)?
++    .await?;
+```
+
+If previously using the `tracing` feature, and using a custom feature set, add
+`tracing` to your list of features.
+
+### Additions
+
+Support for Slash Commands has been added.
+
+The following HTTP requests have been added:
+
+- `create_followup_message`
+- `create_global_command`
+- `create_guild_command`
+- `delete_followup_message`
+- `delete_global_command`
+- `delete_guild_command`
+- `delete_interaction_original`
+- `get_command_permissions`
+- `get_global_commands`
+- `get_guild_command_permissions`
+- `get_guild_commands`
+- `interaction_callback`
+- `set_command_permissions`
+- `set_global_commands`
+- `set_guild_commands`
+- `update_command_permissions`
+- `update_followup_message`
+- `update_global_command`
+- `update_guild_command`
+- `update_interaction_original`
+
+### Enhancements
+
+The following HTTP errors have been added:
+
+- `10063` `UnknownApplicationCommand`
+- `10066` `UnknownApplicationCommandPermissions`
+- `40041` `CommandNameAlreadyExists`
+
+The following HTTP ratelimiting paths have been added:
+
+- `ApplicationCommand(u64)`
+- `ApplicationCommandId(u64)`
+- `ApplicationGuildCommand(u64)`
+- `ApplicationGuildCommandId(u64)`
+- `InteractionCallback(u64)`
+
+The following HTTP routes have been added:
+
+`CreateGlobalCommand { application_id }`
+`CreateGuildCommand { application_id, guild_id }`
+`DeleteGlobalCommand { application_id, command_id }`
+`DeleteGuildCommand { application_id, command_id, guild_id }`
+`DeleteInteractionOriginal { application_id, interaction_token }`
+`GetCommandPermissions { application_id, command_id, guild_id }`
+`GetGlobalCommands { application_id }`
+`GetGuildCommandPermissions { application_id, guild_id }`
+`GetGuildCommands { application_id, guild_id }`
+`InteractionCallback { interaction_id, interaction_token }`
+`SetCommandPermissions { application_id, guild_id }`
+`SetGlobalCommands { application_id }`
+`SetGuildCommands { application_id, guild_id }`
+`UpdateCommandPermissions { application_id, command_id, guild_id }`
+`UpdateGlobalCommand { application_id, command_id }`
+`UpdateGuildCommand { application_id, command_id, guild_id }`
+`UpdateInteractionOriginal { application_id, interaction_token }`
+
+### Changes
+
+`GetReactions::before`, and its corresponding `Route::GetReactionUsers::before`
+has been removed ([#810] - [@7596ff]).
+
+`CreateInvite::{target_user} and {target_user_type}` have been removed [#847] -
```

Comment:
Don't need the {}s here if they're not going to be `use` syntax

Reply:
how did this happen